packages/uhk-web/src/styles/_bootstrap.scss

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Bootstrap v4.5.3 (https://getbootstrap.com/)
+ * Copyright 2011-2020 The Bootstrap Authors
+ * Copyright 2011-2020 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
+*/
+
+@import '~bootstrap/scss/functions';
```

Comment:
Should we need this file? I don't know the uncommented elements contains any sass variables that break the break the current build.

The tree shaker will remove the unused css classes. 
If we keep this file and BS changes the file structure we have to modify this file too. Now I just see an extra file that we have to maintain.

Reply:
We do need it, it's used by a lot of stuff.

Reply:
Tried this both ways. With our _bootstrap.scss stylesheets are 161 kB, with importing whole boostrap they are 189 kB. So maintaining our "just what we need" file is better.